internal/xds/translator/ratelimit.go

```diff
@@ -460,16 +662,28 @@ func buildRateLimitServiceDescriptors(global *ir.GlobalRateLimit) []*rlsconfv3.R
 
 		// Case when both header and cidr match are not set and the ratelimit
 		// will be applied to all traffic.
+		// 3) No Match (apply to all traffic)
 		if !rule.IsMatchSet() {
 			pbDesc := new(rlsconfv3.RateLimitDescriptor)
-			// GenericKey case
-			pbDesc.Key = getRouteRuleDescriptor(rIdx, -1)
-			pbDesc.Value = getRouteRuleDescriptor(rIdx, -1)
+
```

Comment:
would it be easier to reuse `getRouteRuleDescriptor` and return `traffic.name` if `shared: true` ?